routes/api/users.js

```diff
@@ -1,19 +1,35 @@
 const express = require("express");
 const router = express.Router();
 const bcrypt = require('bcryptjs');
-const User = require('../../config/models/User');
+const User = require('../../models/User');
+const passport = require('passport');
+const keys = require('../../config/keys');
+const jwt = require('jsonwebtoken');
+const validateRegisterInput = require('../../validation/register');
+const validateLoginInput = require('../../validation/login');
 
 router.get("/test", (req, res) => res.json({ msg: "This is the users route" }));
 
+router.get('/current', passport.authenticate('jwt', {session: false}), (req,res) => {
+  res.json({
+    id: req.user.id,
+    handle: req.user.handle,
```

Comment:
should be username

Reply:
Changed, please check and finalize if ready